engine/access/rest/websockets/connections.go

```diff
@@ -0,0 +1,57 @@
+package websockets
```

Comment:
Why has this file name been changed to `connections`?

Reply:
[f384b0a](https://github.com/onflow/flow-go/pull/6757/commits/f384b0a24d1165444a57bba1e9eadd09893723a5)